e2e/src/sozu/worker.rs

```diff
@@ -133,7 +133,9 @@ impl Worker {
         println!("Setting up logging");
 
         let server_job = thread::spawn(move || {
-            setup_default_logging(false, "error", &thread_name);
+            if let Err(e) = setup_default_logging(false, "error", &thread_name) {
+                println!("could not setup logging: {e}");
```

Comment:
Use `error!` instead? (same comment for other `[e]print[ln]` calls)

Reply:
but the `error!` needs the logger successfully setup :-)